setup.py

```diff
@@ -27,6 +28,12 @@
     'pytest',
 ]
 
+# WARNING: This imposes limitations on test/requirements.txt such that the
+# full Pip syntax is not supported. See also
+# <http://stackoverflow.com/questions/14399534/>.
+with open(os.path.join('test', 'requirements.txt')) as f:
```

Comment:
I think it does work, but I can't figure out how. :man_shrugging: 

The closest thing I found was [`os.path.normpath`](https://docs.python.org/2/library/os.path.html#os.path.normpath) whose documentation states:

>  Windows, it converts forward slashes to backward slashes.

Note there's [`io.open`](https://github.com/python/cpython/blob/0f48ecddfb79ef541500ecfcbda0b4178af1cc29/Modules/_io/_iomodule.c#L288), but it's not the same as the built-in `open`:

```py
Python 2.7.10 (default, Jul 30 2016, 19:40:32)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import io
>>> io.open
<built-in function open>
>>> open
<built-in function open>
>>> io.open is open
False
```

Neither say anything explicit about calling `os.path.normpath`, and I can't find it in their sources. Even the [Python implementation of `io.open`](https://github.com/python/cpython/blob/0f48ecddfb79ef541500ecfcbda0b4178af1cc29/Lib/_pyio.py#L44) is silent on the issue.